### ISSUE TITLE
Dots in field names break the structure for `multipart/form-data`

### ISSUE THREAD

**Issue by irshadahmad21** [adonisjs/bodyparser#63]
### Package version

v10.0.1

### Describe the bug

For `'application/x-www-form-urlencoded'`, we have an option to pass `queryString: { allowDots: false }` but there is no such option for `'multipart/form-data'` and the fields are parsed using dot for nesting.

We have a form that allows users to submit a URL along with other inputs and files. We need to store some meta related to the submitted URL in an input field like this

```jsx
<input type='hidden' name={`meta[${url}][some_key]`} value={'some value'} />
```

The URL will obviously contain dots.

**Comment by thetutlage** [comment]
Can you share some sample field names for the same?

**Comment by irshadahmad21** [comment]
e.g.

```jsx
<input type='hidden' name={`meta[https://dns.mysite.com][ttl]`} value={'some value'} />
```

**Comment by thetutlage** [comment]
Yeah, seems like it needs a fix. For now (as a workaround), you can wrap the domain inside single quotes and it will work.

```html
<input type="hidden" name="meta['https://dns.mysite.com'][ttl]" value="some value" />
```

**Comment by irshadahmad21** [comment]
> Yeah, seems like it needs a fix. For now (as a workaround), you can wrap the domain inside single quotes and it will work.
> 
> ```
> <input type="hidden" name="meta['https://dns.mysite.com'][ttl]" value="some value" />
> ```

Yeah, that works. Thanks
